website/docs/r/vapp_vm.html.markdown

```diff
@@ -503,6 +503,7 @@ example for usage details.
   using `vcd_org_vdc.vm_placement_policy_ids` (and optionally `vcd_org_vdc.default_compute_policy_id` to make it default).
   In this case, if the placement policy is not set, it will pick the VDC default on creation. It must be set explicitly
   if one wants to update it to another policy (the VM requires at least one Compute Policy), and needs to be set to `""` to be removed.
+* `security_tags` - (Optional; *v3.9+*) List of security tags to be managed by the `vcd_vapp_vm` ressource. Removing `security_tags` will not remove the security tags themselves. They remain unchanged and just stop being managed by `security_tags` in the `vcd_vapp_vm` ressource. This is to keep backwards compatibility with existing security tags that were created by the `vcd_security_tags` ressource.
```

Comment:
nit, just to be more precise (as there's also this `TypeList`):
```suggestion
* `security_tags` - (Optional; *v3.9+*) Set of security tags to be managed by the `vcd_vapp_vm` resource. Removing `security_tags` will not remove the security tags themselves. They remain unchanged and just stop being managed by `security_tags` in the `vcd_vapp_vm` resource. This is to keep backwards compatibility with existing security tags that were created by the `vcd_security_tags` resource.
```

> They remain unchanged and just stop being managed by `security_tags` in the `vcd_vapp_vm` resource

Should we mention what to do to actually delete them? If I understood the conversation with @Didainius, it was `security_tags = [""]`?